### PR TITLE
Tests for services scaling using webhooks

### PR DESCRIPTION
**Pull request by soumyalj**
@sangeethah : Test cases for service scaling using webhooks. Please review. All tests have passed.

### CODE REVIEW

**Comment by sangeethah** [comment]
@soumyalj , Can we add a test case for the following scenario:

scenario 1:
current scale of service is 2.
      "scaleServiceConfig": {
             "action": "up",
             "amount": 3,
             "serviceId": service.id,
             "min": 1,
             "max": 4,
         }
In this case , will service scale to 4  and web hook will report error? Or will the service not scale up at all? 
 
scenario 2:
current scale of service is 4.
      "scaleServiceConfig": {
             "action": "down",
             "amount": 3,
             "serviceId": service.id,
             "min": 2,
             "max": 4,
         }
In this case , will service scale to 2  and web hook will report error? Or will the service not scale down at all?

**Comment by soumyalj** [comment]
@sangeethah : Added the two scenarios. Please review. Will be adding some more tests later.

File: tests/v2_validation/cattlevalidationtest/core/test_webhook.py [hunk 1]
```diff
@@ -0,0 +1,599 @@
...
+    webhook_url = json_resp["url"]
+    webhook_id = json_resp["id"]
+
+    print "Webhook is " + repr(webhook_url)
+    print "Id is " + repr(webhook_id)
+
+    # Execute webhook and ensure the scale cannot be
+    # incremented beyond max scale
+    wh_resp = requests.post(webhook_url)
+    assert wh_resp.status_code == 400
```
Comment (sangeethah) [inline tests/v2_validation/cattlevalidationtest/core/test_webhook.py]: service object needs to be reloaded before checking for scale value.

File: tests/v2_validation/cattlevalidationtest/core/test_webhook.py [hunk 2]
```diff
@@ -0,0 +1,599 @@
...
+    webhook_url = json_resp["url"]
+    webhook_id = json_resp["id"]
+
+    print "Webhook is" + repr(webhook_url)
+    print "Id is" + repr(webhook_id)
+
+    # Execute Webhook
+    wh_resp = requests.post(webhook_url)
+    assert wh_resp.status_code == 200
+    service = client.wait_success(service)
```
Comment (sangeethah) [inline tests/v2_validation/cattlevalidationtest/core/test_webhook.py]: service object needs to be reloaded before checking for scale value.

**Comment by sangeethah** [comment]
LGTM

**Pull request merged**